Projects/DbCLI/CreateDb.py

```diff
@@ -69,84 +70,88 @@
 from rdkit.Chem.MolDb import FingerprintUtils
 
 # ---- ---- ---- ----  ---- ---- ---- ----  ---- ---- ---- ----  ---- ---- ---- ----
-from optparse import OptionParser
-parser = OptionParser(_usage, version='%prog ' + _version)
-parser.add_option('--outDir', '--dbDir', default='', help='name of the output directory')
-parser.add_option('--molDbName', default='Compounds.sqlt', help='name of the molecule database')
-parser.add_option('--molIdName', default='compound_id', help='name of the database key column')
-parser.add_option('--regName', default='molecules', help='name of the molecular registry table')
-parser.add_option('--pairDbName', default='AtomPairs.sqlt', help='name of the atom pairs database')
-parser.add_option('--pairTableName', default='atompairs', help='name of the atom pairs table')
-parser.add_option('--fpDbName', default='Fingerprints.sqlt',
+import argparse
+
+parser = argparse.ArgumentParser(usage)
+
+parser.add_argument('filename')
```

Comment:
Check the argparse documentation if this defines a required or an optional argument. What you want here is an optional argument.

Reply:
Yeah, realized that. Shall work on it.